vignettes/howto-dm-copy.Rmd

```diff
@@ -97,16 +97,16 @@ stopifnot(grepl(remote_name_total_loans, sql_render(my_dm_total_computed$total_l
 ```
 
 Note the differences in queries returned by `sql_render()`.
-`my_dm_total$total_loans` is still being lazily evaluated and the full query constructed from the chain of operations that generated it, and is required to run to access it, is still in place.
-Contrast that with `my_dm_total_computed$total_loans` where the query has been realized and accessing its rows requires a simple `SELECT *` statement.
+`my_dm_total$total_loans` is still being lazily evaluated and the full query constructed from the chain of operations that generated it is still in place and needs to be run to access it.
+Contrast that with `my_dm_total_computed$total_loans`, where the query has been realized and accessing its rows requires a simple `SELECT *` statement.
 The table name, `` `r remote_name_total_loans` ``, was automatically generated as the `name` argument was not supplied to `compute()`.
 
-The default is to create a temporary table.
-If you want results to persist across sessions in permanent tables, `compute()` must be called with the argument `temporary = FALSE` and a table name for the `name` argument.
+The default is to create a **temporary** tables.
+If you want results to persist across sessions in **permanent** tables, `compute()` must be called with the argument `temporary = FALSE` and a table name for the `name` argument.
 See `?compute` for more details.
 
-When called on a whole dm object (without zoom), `compute()` materializes all tables into new (temporary or persistent) tables by executing the associated SQL query and storing the full results.
-Depending on the size of your data this may take considerable time or be infeasible.
+When called on a whole `dm` object (without zoom), `compute()` materializes all tables into new (temporary or persistent) tables by executing the associated SQL query and storing the full results.
+Depending on the size of your data, this may take considerable time or may even be unfeasible.
```

Comment:
unfeasible is gaining traction, but infeasible doesn't seem to be wrong: https://books.google.com/ngrams/graph?content=infeasible%2Cunfeasible&year_start=1800&year_end=2019&corpus=26&smoothing=3&direct_url=t1%3B%2Cinfeasible%3B%2Cc0%3B.t1%3B%2Cunfeasible%3B%2Cc0

Is this a UK/US thing?